auth/auth/driver/driver.py

```diff
@@ -315,6 +315,7 @@ async def create(self, name):
         self.name = name
 
     async def _delete(self, name):
+        assert namespace != 'default'
```

Comment:
I don't see where namespace is defined. I think you meant `HAIL_DEFAULT_NAMESPACE`. Also, should we assert the same thing for `create`?

Reply:
Oops, I think that got lost in the rebase. I'll add that though.